### PR TITLE
Enhance configuration options

### PR DESCRIPTION
**Pull request by michaljemala**
**Description of the change**
I would like to this image with Google Cloud SQL for Postgres. The configuration is in many ways similar to what is described in the [docs for AWS Aurora](https://www.pgpool.net/docs/latest/en/html/example-aurora.html). In order to allow changing/setting these configuration parameters easily I have extended the image as follows:
1. Add ability to disable SR checks, i.e. expose new env. variable `PGPOOL_HEALTH_CHECK_PERIOD`
2. Add ability to configure statement levels load-balancing, i.e. expose new env. variable `PGPOOL_ENABLE_STATEMENT_LOAD_BALANCING `
3. Separate SR check user from health check user
4. Add ability to configure health check parameters, i.e. adding new env. variables
`PGPOOL_HEALTH_CHECK_PERIOD`, `PGPOOL_SR_CHECK_USER` `PGPOOL_HEALTH_CHECK_TIMEOUT`, `PGPOOL_SR_CHECK_PASSWORD`, `PGPOOL_HEALTH_CHECK_USER`, `PGPOOL_HEALTH_CHECK_PASSWORD`, `PGPOOL_HEALTH_CHECK_MAX_RETRIES`, `PGPOOL_HEALTH_CHECK_RETRY_DELAY`

**Benefits**

Ability to tweak the config without overriding the whole configuration file.

**Possible drawbacks**

Do not know of any at the moment.

**Applicable issues**

Do not know of any at the moment.

**Additional information**

For backward compatibility `PGPOOL_SR_CHECK_USER` is set as a default value for the `PGPOOL_HEALTH_CHECK_USER`.

### CODE REVIEW

File: 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh [hunk 1]
```diff
@@ -293,6 +310,7 @@ EOF
 pgpool_create_config() {
     local -i node_counter=0
     local load_balance_mode=""
+    local statement_level_load_balance=""
```
Comment (juan131) [inline 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:313]:
```suggestion
    local statement_level_load_balance="off"
```

File: 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh [hunk 2]
```diff
@@ -303,6 +321,12 @@ pgpool_create_config() {
         load_balance_mode="off"
     fi
 
+    if is_boolean_yes "$PGPOOL_ENABLE_STATEMENT_LOAD_BALANCING"; then
+        statement_level_load_balance="on"
+    else
+        statement_level_load_balance="off"
+    fi
+
```
Comment (juan131) [inline 4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh:329]:
```suggestion
is_boolean_yes "$PGPOOL_ENABLE_STATEMENT_LOAD_BALANCING" && statement_level_load_balance="on"
```

**Comment by michaljemala** [comment]
Thanks a lot for the review and suggestions for improvements. I have addressed your requests.

**Comment by michaljemala** [comment]
Thanks @juan131 for prompt review 👍🏻

**Comment by juan131** [comment]
@michaljemala a new version was released including your changes (version **4.1.1-debian-10-r83**). Please give it a try

**Pull request merged**